.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: python
+python:
+  - "2.7"
```

Comment:
```suggestion
  - "3.7"
```

Reply:
but will it blend?

Reply:
?

Reply:
https://github.com/RoboticsBrno/RB3204-RBCX/runs/646119840